eos-tech-support/eos-write-live

```diff
@@ -0,0 +1,272 @@
+#!/bin/bash -e
```

Comment:
This might be better named `eos-write-live-image`. `write-live` is a bit ambiguous to me.